package.json

```diff
@@ -17,8 +17,8 @@
     "start-dev": "npm-run-all --parallel watch start",
     "build": "npm-run-all build:*",
     "build:font-awesome": "node scripts/build-fontawesome.js",
-    "build:webpack": "webpack",
-    "watch": "webpack -w",
+    "build:webpack": "webpack -p",
+    "watch": "webpack -d --watch",
```

Comment:
For whoever is wondering, `node_modules/.bin/webpack -h` says:

```
  -d    shortcut for --debug --devtool sourcemap --output-pathinfo
  -p    shortcut for --optimize-minimize
```

I see this as unofficial development/production flags, am I correct?
If so, it's a bit confusing that `build:webpack` implies production and `watch` implies development. I know I sometimes use `npm run build:webpack` when I just want a single build without a FS watch.

IMO (but MO is fairly Webpack-inexperienced 😅), I think we should have the commands be environment-independent, and the configuration files behave differently depending on the environment (or select different files depending on environment, if the configurations are too disjoint).
Is that stupid?